dts/bindings/radio_fem/nordic,nrf21540-fem.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
```

Comment:
Isn't that a duplication of that https://github.com/zephyrproject-rtos/zephyr/commit/19a93a3c68c79dd920678f38fbfcfd66fe32afee ?

Reply:
Yes. Good point. I remove it from this pr